### ISSUE TITLE
2024 accessibility improvements

### ISSUE THREAD

**Issue by timwaters** [timwaters/learninginleeds#114]
Tracking new accessibility issues. Split out items into new issues if necessary

non code issues

- [ ] homepage - alt text for news item - pathways to higher education for adult learners' image
- [x] about - The logos in the page don't have alt text. Considering they are not decorative they should have descriptive alt text. E.g. 'West Yorkshire Combined Authority logo' (changed via admin panel)

main issues

- [x] validations - ensure topic, news, story has alt text
- [x] Data tables that contain both row and column headers use the scope attribute, or id and headers, to identify cells.
- [x] some text on navigation description contrast  
- [x] Search icon in mobile menu using `<i>` element and should not be used. Change to another element such as span
- [x] Twitter link Adjust the aria-label to not include the url and instead use human readable text e.g. Visit inclusive growth leeds Twitter
- [x] add sitemap and link in footer
- [x] #115
- [x] Slider arrows alt text should be more descriptive (we use  aria label like "next slide")
- [x] change focus order of slider, prev then slider, then next
- [x] slider Screen readers don't announce the content within each slider. This is caused by the aria-label on the <a> tag. Remove this and all content within it will be read out.
- [x] put skip to main content above menu
- [x] hamburger menu - The mobile menu when opened has an invalid aria attribute. Aria-expanded is not allowed on div's. change the role or element. (Patched bootstrap for this. see: 05f8b1ca25e3b )
- [x] fix topic icons links. The icons used for topic are wrapped in links as well as the text below. Either remove the link around the image, or wrap the link around the whole element. (We wrapped around whole element
- [x] change from "near" to "location". the hint text needs linking to the input so it's read after the label. (Also for course directions)  [ hint text is linked via aria-describedby ] 
- [x] #117
- [x] courses list - Add aria-describedby and link to the 'sorted by'
- [x] course pages - remove title, use course title as title (h1 should be descriptive)
- [x] add nav arrows / control to map
- [x] read more (stories / news) The use of 'read more' is not recommended. Make links more descriptive and related to their content or add aria-describedby and link to the heading

optional ones
- [x] optional - on submission of postcode on page load the focus should be on the journey finder block (currently, loaded page goes to target #directions)
- [x] optional - The maps on course pages need an aria label so not everything is read out in sequence every time. E.g. id="map" aria-label="Map showing X location"
- [ ] #116
